.lintstagedrc.cjs

```diff
@@ -1 +0,0 @@
-module.exports = require('@1stg/lint-staged/tsc')
```

Comment:
No lint-staged anymore?

Reply:
Correct. I don’t believe it is needed. Other projects maintained by unified don’t use them either.

Reply:
As above.

Reply:
As above.